app/liquid/liquid_renderer.rb

```diff
@@ -11,7 +11,9 @@ def initialize(page, layout: nil, request_country: nil, member: nil, url_params:
   end
 
   def render
-    template.render( data ).html_safe
+    Rails.cache.fetch(cache_key) do
+      template.render( data ).html_safe
+    end
```

Comment:
I've put a splurged at the bottom of `production.rb`:
```ruby
  # config.action_controller.perform_caching = true
  #
  # We don't want to use memory_store, it's just a convenient choice for development.
  # AWS provides a managed memcached/redis service, with a t2.micro instances (1Gb) as
  # part of their free tier.
  #
  # config.cache_store = :memory_store, { size: 10.megabytes }
```

We'll only want to cache in production, but for QA just copy the above into `development.rb`